README.md

```diff
@@ -21,39 +21,32 @@ Do the following steps to start sending Knot.x application metrics to Graphite:
 ```json
     {
       "groupId": "io.knotx",
-      "artifactId": "metrics-sender",
+      "artifactId": "knotx-dashboard",
       "version": "X.Y.Z",
       "included": true
     }
 ```
 * Run `bin\knotx resolve` to resolve new dependencies and add `metrics-sender` to the instance classpath
-* Add `metrics-sender` entry to `application.conf` modules list
+* Add `metricSender` entry to `application.conf` modules list
 ```
 "metricsSender=io.knotx.metrics.SenderVerticle"
 ```
-* Copy `conf/dashboardStack.conf` to application `conf` folder
-* Add `dashboardStack.conf` to application stores defined in `bootstrap.json`, e.g.
-```json
-...
-    "stores": [
-      ...
-    
-      {
-        "type": "file",
-        "format": "conf",
-        "config": {
-          "path": "${KNOTX_HOME}/conf/dashboardStack.conf"
-        }
-      }
-    ]
-...
+* Add `metricSender` entry to `application.conf` modules configuration
+```
```

Comment:
Why no longer use metrics as a separate store?
Is there something wrong with this approach?

Reply:
I thought we no longer modified bootstrap.json, but included configuration files in application.conf. Is that correct?

Reply:
It is more clear to me to define metrics as separate store entry.
Metrics does not introduce any business logic into Knot.x instance.

Reply:
Ok, reverted this change.